services/API-service/src/scripts/seed-admin-area-data.ts

```diff
@@ -85,38 +85,135 @@ export class SeedAdminAreaData implements InterfaceScript {
 
     // UGA
     if (envCountries.includes('UGA')) {
+      const countryCodeISO3 = 'UGA';
+      const adminlevel = 2;
       // Flood vulnerability
       const floodVulnerabilityFilename = `./src/scripts/git-lfs/admin-area-data/flood_vulnerability_UGA.csv`;
       const floodVulnerabilityData = await this.seedHelper.getCsvData(
         floodVulnerabilityFilename,
       );
       const floodVulnerabilityDataArray = floodVulnerabilityData.map(area => {
         return {
-          countryCodeISO3: 'UGA',
-          adminLevel: 2,
+          countryCodeISO3: countryCodeISO3,
+          adminLevel: adminlevel,
           placeCode: area['placeCode'],
           indicator: area['indicator'],
           value: area['value'],
         };
       });
-
       await this.adminAreaDataRepository.save(floodVulnerabilityDataArray);
 
       // COVID risk
       const covidRiskFilename = `./src/scripts/git-lfs/admin-area-data/covid_risk_UGA.csv`;
       const covidRiskData = await this.seedHelper.getCsvData(covidRiskFilename);
       const covidRiskDataArray = covidRiskData.map(area => {
         return {
-          countryCodeISO3: 'UGA',
-          adminLevel: 2,
+          countryCodeISO3: countryCodeISO3,
+          adminLevel: adminlevel,
           placeCode: area['placeCode'],
           indicator: area['indicator'],
           value: area['value'],
         };
       });
-
       await this.adminAreaDataRepository.save(covidRiskDataArray);
     }
+
+    // ETH
+    if (envCountries.includes('ETH')) {
```

Comment:
It's not advisable to have country-specific code blocks. It's not an issue yet but seems like a refactor waiting to happen.